OSD-2132 Add Redmine workload

mrbarge opened the pull request:
In support of OSD-2132 to improve customer app testing, this PR adds a new workload to build and host the Redmine web application via a Kubernetes deployment, using persistent storage for both the backend MySQL database service and the frontend web service, and secrets to manage the database credentials.

An initial validation method has been written to check that the service is responding. The deployment should complete within several minutes.

It has been successfully tested on staging and prod CI clusters so far.

If accepted as an additional workload, more sophisticated validation will be added as part of OSD-2132.

Thank you! :)

meowfaceman commented:
/approve
This is awesome. It looks like the failure in doTest is a linter issue -- did you mean for that to be a log.Printf instead of an assignment to err? Either way, once that's in I think this is looking good.

mrbarge commented:
> /approve
> This is awesome. It looks like the failure in doTest is a linter issue -- did you mean for that to be a log.Printf instead of an assignment to err? Either way, once that's in I think this is looking good.

Thanks @meowfaceman ! I've addressed that issue now and updated the branch accordingly. Cheers for the review.

jeefy commented:
/lgtm

Awesome work!
